### ISSUE TITLE
No visible resources after migrating to 2.6.0

### ISSUE THREAD

**Issue by vjrj** [gbif/ipt#1835]
It seems that the home index have some issue with quotes or similar that prevents the home to visibilize our resources:

![image](https://user-images.githubusercontent.com/180085/193053983-ae90e0e3-fef8-4f74-a0cb-91429939c756.png)

![image](https://user-images.githubusercontent.com/180085/193053991-0d2d3534-f9b5-485e-943d-5c104c8e4835.png)

I just downgraded to ipt `2.5.8`.

The same happened with https://ipt-demo.gbif.es but we removed a pair of testing resources to solve it, something that we cannot do in production.

A problematic resource, if helps:
https://datos.gbif.es/others/failed-resource.tgz

**Comment by mike-podolskiy90** [comment]
Thank you @vjrj 
We already spotted that on our cloud IPT, fix already committed, release coming soon

**Comment by vjrj** [comment]
Thanks @mike-podolskiy90 !

**Comment by mike-podolskiy90** [comment]
https://github.com/gbif/ipt/commit/270562e0e8b9ff9db5c0921318eea62a37232185

**Comment by vjrj** [comment]
Applying this commit does not solve the issue for us, see:
https://ipt.gbif.es

**Comment by vjrj** [comment]
It has newlines !??
![image](https://user-images.githubusercontent.com/180085/193058935-a1b5e9a3-29f0-4a57-889d-39efc0c6df03.png)

**Comment by mike-podolskiy90** [comment]
Yes, because of issues in the metadata. Keywords should separated by semicolon. Fix must've helped though, checking

**Comment by mike-podolskiy90** [comment]
@vjrj I corrected a regex

**Comment by vjrj** [comment]
it works now https://ipt.gbif.es/ thanks indeed @mike-podolskiy90

**Comment by ckotwn** [comment]
Awaiting the update/fix for Asia IPT ;)

**Comment by vjrj** [comment]
Hi @ckotwn as a workaround, I edited this file:

`/var/lib/tomcat9/webapps/ipt/WEB-INF/pages/macros/resourcesTable.ftl`

(it can be in a different location in your side) and changed the line 53 
`'${r.eml.subject!}'`
with:
`'${(r.eml.subject?replace("[\r\n]+", "<br>", "r")?replace("\'", "\\'")?replace("\"", '\\"'))!}'`

as in the commit https://github.com/gbif/ipt/commit/dea9fef2766e0cf68004992b7bcd5563aa67dbe8

**Comment by ckotwn** [comment]
Thanks @vjrj for the tip! Although I don't have root access to the Asia IPT instance guess this is beyond my reach ;)

**Comment by mike-podolskiy90** [comment]
No need for workarounds, I've just released the 2.6.1
https://repository.gbif.org/content/groups/gbif/org/gbif/ipt/2.6.0/ipt-2.6.0.war
